lib/experimental/html/class-wp-html-tag-processor.php

```diff
@@ -961,7 +961,29 @@ public function set_attribute( $name, $value ) {
 		if ( true === $value ) {
 			$updated_attribute = $name;
 		} else {
-			// @TODO: What escaping and sanitization do we need here?
+			/**
```

Comment:
this looks like a docblock comment describing the `$escaped_new_value` variable except it doesn't that. if we want to attach the comment maybe we should attach it to the `set_attributes()` function; if we only want to explain then it shouldn't be a docblock I think.

```php
/*
 * Prevents…
```

Reply:
I moved this part to the function docblock.